c_common/models/compressors/src/bit_field_common/routing_tables.h

```diff
@@ -67,7 +67,8 @@ entry_t* routing_tables_append_get_entry(void) {
     // check that we're not hitting the max entries supported by the table
     if (multi_table.n_entries == (int) multi_table.max_entries) {
         log_error(
-                "there is no more space in this multi-table for this entry.");
+                "there is no more space out of %d entries in this multi-table"
+                "for this entry.", multi_table.max_entries);
```

Comment:
I've noticed this more in various places recently - grammatically we should be starting error / log messages such as this with a capital letter.  Probably worth going through all errors / warnings etc. and checking grammar / spelling etc. before we go ahead with a release...

Reply:
Fixed